app/src/main/java/com/seay/game_of_thrones/activity/MainActivity.java

```diff
@@ -43,24 +53,21 @@ protected void onCreate(Bundle savedInstanceState) {
 
```

Comment:
I can't mark the exact spot, but can you delete the white space after `onCreate()`